### PR TITLE
351548 - add filters on ExternalAppRecord sync with Category

### PR DESCRIPTION
**Pull request by nicolas-brousse**

### CODE REVIEW

File: app/controllers/external_app_records_controller.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def index
     @external_app_records_count = @external_app_records.count
     @filter = ProcessorFilter.new(@external_app_records, params)
     @external_app_records = @filter.results
-    @servers_count = Server.no_pdus.count
+    @servers_count = Server.glpi_synchronizable.count
```
Comment (B-Rass) [inline app/controllers/external_app_records_controller.rb:9]: To be removed (not used anymore)

File: app/views/categories/index.html.erb [hunk 2]
```diff
@@ -27,6 +27,10 @@
       <% end %>
     <% end %>
 
+    <% table.with_column(Category.human_attribute_name(:is_glpi_synchronizable), sort_by: :is_glpi_synchronizable, class: "text-center", style: "min-width: 240px; width: 240px") do |category| %>
```
Comment (B-Rass) [inline app/views/categories/index.html.erb:30]:
```suggestion
    <% table.with_column(Category.human_attribute_name(:is_glpi_synchronizable), sort_by: :is_glpi_synchronizable, class: "text-center", style: "width: 240px") do |category| %>
```

Comment (B-Rass) [reply app/views/categories/index.html.erb:30]: `min-width` is not useful here

File: db/migrate/20241204143353_add_is_glpi_synchronizable_to_categories.rb
Comment (B-Rass) [inline db/migrate/20241204143353_add_is_glpi_synchronizable_to_categories.rb:1]: We talked about putting all the current categories to true. But we can let it that way

**Review by B-Rass — CHANGES_REQUESTED** [review]

File: app/controllers/external_app_records_controller.rb (hunk 1 above)
Comment (nicolas-brousse) [reply app/controllers/external_app_records_controller.rb]: Done

File: db/migrate/20241204143353_add_is_glpi_synchronizable_to_categories.rb
Comment (nicolas-brousse) [reply db/migrate/20241204143353_add_is_glpi_synchronizable_to_categories.rb:1]: I've memorized the opposite 😅

File: app/views/categories/index.html.erb (hunk 2 above)
Comment (nicolas-brousse) [reply app/views/categories/index.html.erb]: Done

**Review by B-Rass — APPROVED** [review]

**Pull request merged**